### DISCUSSION TITLE
Create PySWMM Configuration "Standard Method"

### DISCUSSION THREAD

**Discussion by bemcdonnell** [pyswmm/pyswmm#448] [General]
I wanted to propose that we consider developing a standard method around configuring the workflow around using PySWMM.  As more and more municipalities, academics, and consulting firms join and use, we might propose a standard process of organizing the model, the support files, and the actual Python module.  By doing this, it will create a space to enable easier integrations.   Before proposing what I have in mind, I just wanted to gather some opinions first.  Particularly @karosc, @mehmetbercan ,  @abhiramm7, @aerispaha, and @jackieff.  Do you have any opinions?

**Comment by aerispaha** [comment]
I think a standard/recommended workflow is a good idea. I feel like we can demonstrate this in a series of examples, maybe capturing a few different workflow flavors. 

One thing I think we should consider is leveraging Jupyter notebooks - this might not be the gold standard approach, but I think that development paradigm is good for experimentation, especially for relatively newer Python users. This could make it easier for new users to on-board.
